### ISSUE TITLE
Error for GraphServiceClient

### ISSUE THREAD

**Issue by KEFIRAEnterprise** [microsoftgraph/msgraph-sdk-python#228]
client = GraphServiceClient(credentials=credential, scopes=scopes)

TypeError: GraphServiceClient.__init__() got an unexpected keyword argument 'credentials'

**Comment by derekpurdy** [comment]
I needed to do the following to get it to work, referenced from https://learn.microsoft.com/en-us/graph/sdks/create-client?tabs=Python

```
credential = DefaultAzureCredential()
auth_provider = AzureIdentityAuthenticationProvider(credential)

adapter = GraphRequestAdapter(auth_provider)
graph_client = GraphServiceClient(adapter)
```

**Comment by tikicoder** [comment]
Further discussion has happened here
https://github.com/microsoftgraph/msgraph-sdk-python/issues/246

**Comment by samwelkanda** [comment]
@KEFIRAEnterprise 

We apologize for that. Our README contained changes for a version that had not been published to PyPI. Please upgrade to the now available sdk version `1.0.0a13`

Thanks.

**Comment by samwelkanda** [comment]
Closing for now. Feel free to re-open if you have further questions.
